backend/internal/process/process.go

```diff
@@ -265,11 +272,18 @@ func (m *ProcessManager[Meta]) Spawn(procConfig ProcessConfig[Meta]) (*Process[M
 		Meta:      procConfig.Meta,
 	}
 
+	// Reap zombies
+	go entry.waitForExit(entry.Pid)
```

Comment:
It is important to start the reaper immediately, because if anything fails, we want to be reaping still